extensions/aws/aws-test/src/testFixtures/java/org/eclipse/dataspaceconnector/aws/testfixtures/annotations/AwsS3IntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.aws.testfixtures.annotations;
```

Comment:
nit: "fixtures" instead of "testfixtures"?

Reply:
Module itself is named as testfixtures. So kept it similar.